spec/models/manageiq/providers/redhat/infra_manager/refresher_3_0_spec.rb

```diff
@@ -246,7 +246,7 @@ def assert_specific_vm_powered_on
       :standby_action        => nil,
       :connection_state      => "connected",
       :cpu_affinity          => nil,
-      :memory_reserve        => nil,
+      :memory_reserve        => 715_128_832,
```

Comment:
If possible, in the tests, can you use a Rails helper method to make it clearer what this number represents?  For example, this one can be `682.megabytes` which reads a lot nicer and is probably easier to correlate back to the source of the refresher.